index.html

```diff
@@ -0,0 +1,30 @@
+<html>
+<head>
+	<title>Famous Person Website</title>
+	<link rel="stylesheet" type="text/css" href="css/styles.css">
+</head>
+
+<body>
+
+	<nav>
+	About |
+	<a href="discography.html">Discography</a>
+	</nav>
+
+	<h1>Caetano Veloso</h1>
+	<p>
+	<img src="images/caetano-veloso-01.gif" alt="Profile"> 
+	</p>
+
+	
+	<h3>Artist Biography</h3>
+
+	<p>
+	Caetano Emanuel Viana Telles Veloso, born August 7, 1942, better known as Caetano Veloso, is a Brazilian composer, singer, guitarist, writer, and political activist. Veloso first became known for his participation in the Brazilian musical movement Tropicalismo, which encompassed theatre, poetry and music in the 1960s, at the beginning of the Brazilian military dictatorship. He has remained a constant creative influence and best-selling performing artist and composer ever since. Veloso has won nine Latin Grammy Awards and two Grammy Awards. On November 14, 2012, Veloso was honored as the Latin Recording Academy Person of the Year.
+
```

Comment:
:hammer: Since this content is inside a `<p>` tag, html will not "see" the blank line here. While this won't break anything, if you intend on putting a blank line on the page, you'd have to make another `<p>` tag for the second line. Otherwise, make sure you remove these unnecessary blank lines from your code.